e2e/core/viewer/viewer-content-services-component.e2e.ts

```diff
@@ -318,7 +298,7 @@ describe('Content Services Viewer', () => {
 
             await viewerPage.checkZoomInButtonIsNotDisplayed();
             await viewerPage.checkUnknownFormatIsDisplayed();
-            await expect(await viewerPage.getUnknownFormatMessage()).toBe('Couldn\'t load preview. Unknown format.');
+            await expect(await viewerPage.getUnknownFormatMessage()).toBe("Couldn't load preview. Unknown format.");
```

Comment:
do we need await before expect?

Reply:
it's a separate ESLint rule to introduce, we have 1000+ `await expect` incorrectly used in tests